solace-java-sample-app/src/main/resources/application.properties

```diff
@@ -1 +1,3 @@
 solace.java.host=192.168.164.167
+solace.java.reapplySubscriptions=true
+solace.java.messageAckMode=client_ack
```

Comment:
I don't think you need to add these new properties here. It's good enough to add them to the readme.